src/App/components/FileSelector.tsx

```diff
@@ -43,7 +50,7 @@ const FileSelector = (prop: {
 FileSelector.propTypes = {
   save: PropTypes.bool,
   buttonText: PropTypes.string,
-  buttonIcon: PropTypes.object,
+  buttonIcon: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
This is better than what I've attempted in #216 - let's do it your way.

Reply:
Ok